app/components/activity_notifications/activity_notifications.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= c 'card', append_class: 'ActivityNotifications' do %>
+  <%= c 'heading', tag: :h2 do %>
+    <%= t '.card_heading' %>
+  <% end%>
+  <% if activity_notifications.any? %>
+      <ul role='list' class='ActivityNotifications-list'>
+        <% activity_notifications.each do |notification_hash| %>
+          <li class='ActivityNotifications-listItem' >
+            <%= c 'avatar', record: notification_hash[:record] %>
+            <header class='ActivityNotifications-listItemHeader'>
+              <p>
+                <%= notification_hash[:group_message] %>
+              </p>
+              <time class='ActivityNotifications-listItemRelativeTime'>
+                <%= "vor #{time_ago_in_words(notification_hash[:created_at])}" %>
```

Comment:
I did that at one point, but reverted it because I don't know enough German to know if we would ever use `etwa eine Stunde` for any reason without the `vor`. Also, it should be noted that these translations come from Rails, they were not added by us, so that made me think there might be instances where we would. The final consideration that helped me make the decision in favor of leaving it here in the template was that it would be repeated ~20x in the `de.yml` vs only once here.

For now, I'll just leave it for those reasons :point_up:, but I don't have a strong feeling about it so let me know if these are things you already considered and would still prefer it in the translation yml. One strong argument would be that if we use another language, this would not work, for example.